### PR TITLE
Symlink pages/enclone_css_v2.css to enclone_core/src/

### PR DESCRIPTION
**Pull request by sjackman**
This change is needed for some work that Adam is doing for remote Bazel builds.

### CODE REVIEW

**Comment by DavidBJaffe** [comment]
OK cool, are you quite certain that this will not break existing HTML headers that live outside 10x?

**Review by adam-azarchs — APPROVED** [review]

**Comment by DavidBJaffe** [comment]
I am uncomfortable with this change, for two reasons: (1) the file is currently in a logical place, relative to the overall repo structure; (2) I'm not sure how we would test to make sure that browsers correctly handle the link.

Is this pretty much the last change that's needed?  If so I can noodle on it more.

**Comment by adam-azarchs** [comment]
It should be, at least for now.

The problem is that the build for a crate isn't allowed to depend on files outside of the crate directory.  What we're planning on doing is switching (at least for the bazel build) to using `cargo vendor` to get dependencies.  This is a much cleaner solution in a lot of ways, and absolutely necessary to unblock our future remote build strategy (which we really, really need because having everyone build from the same place isn't scalable).  However it means that things are incorporated into the build as if they were packaged crates, even when they're actually being fetched from `git`.  This means that
1. `git` commands won't work because there's no `.git` directory in the packaged crate.
2. `include` or `exclude` directives in `Cargo.toml` are honored, rather than having hacky heuristics for what files to exclude from the build.
3. Relative paths outside of the crate directory will _not_ work.  Which is to say, `path`-based dependencies in `Cargo.toml` will still work (they'll be converted appropriately) but e.g. `include_str!` will not.

Basically, if it would be broken as a published crate (other than the requirement that published crates should only depend on other published crates) it would be broken under vendoring as well.

**Comment by adam-azarchs** [comment]
It actually appears that putting the symlink in `enclone_core/src` would also work, as vendoring appears to take care of desymlinking it.

**Comment by sjackman** [comment]
Symlinked `pages/enclone_css_v2.css` to `enclone_core/src/`

**Comment by DavidBJaffe** [comment]
OK that looks good, thank you!
